src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+const convertToObject = (sourceString) => {
+  const splittedSource = sourceString.split(';');
+  const resultObj = {};
+
+  for (const element of splittedSource) {
+    const resultArray = element.replace(/[[\n\t]/g, '').split(':');
```

Comment:
The regular expression used in the 'replace' method seems to have a typo. It should be /[
	]/g to match newlines and tabs. The current expression with double brackets [[] is incorrect and may not work as intended.